lib/manager/cloudbuild/readme.md

```diff
@@ -0,0 +1 @@
+The `cloudbuild` manager extracts `docker` datasources from Cloud Build config files.
```

Comment:
Add a link here to a page which describes the configuration files

Reply:
Added.